R/check_factorstructure.R

```diff
@@ -0,0 +1,127 @@
+#' Check for suitability of data for Factor Analysis (FA)
+#'
+#' This checks whether the data is appropriate for Factor Analysis (FA) by running the \link[=check_sphericity]{Bartlett's Test of Sphericity} and the \link[=check_kmo]{Kaiser, Meyer, Olkin (KMO) Measure of Sampling Adequacy (MSA)}.
+#'
+#' @inheritParams check_kmo
+#' @examples
+#' library(parameters)
+#'
+#' check_factorstructure(mtcars)
+#'
+#' @seealso check_kmo check_sphericity
+#' @export
+check_factorstructure <- function(x, ...){
```

Comment:
@strengejacke however, this way of simply running the two gives an ugly output (as they are not separated). I tried capturing the output, but then the colour disappears... any tips to address that?

Reply:
Give me a short update, what do you want to achieve, and where's the current problem?

Reply:
- I am slowly adding a new set of features in parameters around dimensionality reduction (including support for PCA / FA etc.)
- In this context, I added the two popular tests for checking if the data is suitable for FA
- I would like to have a master function that returns the output of these two checks "at once"
- A function simply running these two checks gives an ugly collated output
- how to improve it?

Reply:
Ok, I'll look into this later today.